st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -16,6 +16,7 @@
 import sys
 import json
 import argparse
+import logging as stdlib_logging
```

Comment:
I forgot I had done that. It's from a quick attempt at getting this to log in the same file as the action, `_set_up_logger` is a copy and paste from the Action class. I didn't intend to submit with that code. (This is also the answer to @manasdk's question about `_set_up_logger`.)

Do you have a suggestion on how to accomplish the same goal? Assuming logging into the same files is desired. 

As to why it's being passed in, in SensorService it was using the logger from the instance of SensorWrapper that gets passed in. I wanted to avoid changing SensorService's behavior.